### ISSUE TITLE
The link to share (and store) result is not working

### ISSUE THREAD

**Issue by adrientaudiere** [GreenAlgorithms/green-algorithms-tool#41]
Hi, 
Thank you for this very helpful tool. It seems that the link to share result is broke. For my configuration, here is the [link](
http://calculator.green-algorithms.org//?runTime_hour=12&runTime_min=0&appVersion=v2.2&locationContinent=Europe&locationCountry=France&locationRegion=FR&coreType=CPU&numberCPUs=6&CPUmodel=Core%20i5-9400&memory=64&platformType=personalComputer&PSFradio=Yes&PSF=2) I obtained. When I open this link, the right part (parameters) is gray and unchangeable. 

I tried using firefox and google chromium under Debian 12.7.

**Comment by lattaai8** [comment]
Hello, I tried to solve the issue.

This is what I did:

Modified the fillInFromURL function to allow editing of values from shared links. Removed the disabling of input fields when URL parameters are present.

You can review changes in this commit: https://github.com/lattaai8/GreenAlgorithms-green-algorithms-tool-41/commit/f504d21e5ff2fb615b90585d2983458f5962f13b.


> [!CAUTION]
> Disclaimer: The commit was created by Latta AI and you should never copy paste this code before you check the correctness of generated code. Solution might not be complete, you should use this code as an inspiration only.

---
This issue was tried to solve for free by Latta AI - https://latta.ai/ourmission

If you no longer want Latta AI to attempt solving issues on your repository, you can block this account.

**Comment by Llannelongue** [comment]
Yes indeed the method with the URL seemed to be too clunky to work reliably. It took some time, but we've now fixed it in the big release coming later this week. You'll now have the option to share (or import) results using a csv.
Closed by #46
